documentation/site/content/samples/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -172,358 +173,193 @@ wlsregcred                                kubernetes.io/dockerconfigjson   1
-   The persistent volume claim wls-azurefile-1683777168 already exists in NameSpace default
-   configmap/domain1-create-weblogic-sample-domain-job-cm created
-   Checking the configmap domain1-create-weblogic-sample-domain-job-cm was created
-   configmap/domain1-create-weblogic-sample-domain-job-cm labeled
-   Checking if object type job with name domain1-create-weblogic-sample-domain-job exists
-   No resources found in default namespace.
-   Creating the domain by creating the job /home/username/azure/weblogic-domains/domain1/create-domain-job.yaml
-   job.batch/domain1-create-weblogic-sample-domain-job created
-   Waiting for the job to complete...
-   status on iteration 1 of 30
-   pod domain1-create-weblogic-sample-domain-job-v9hp6 status is Init:0/1
-   status on iteration 2 of 30
-   pod domain1-create-weblogic-sample-domain-job-v9hp6 status is Completed
-   domain.weblogic.oracle/domain1 created
-   cluster.weblogic.oracle/domain1-cluster-1 created
-
-   Domain domain1 was created and will be started by the WebLogic Kubernetes Operator
-
-   Administration console access is available at http://wlsaks1683-wlsresourcegroup-260524-3dtnmx4n.hcp.eastus.azmk8s.io:30701/console
-   The following files were generated:
-     /home/username/azure/weblogic-domains/domain1/create-domain-inputs.yaml
-     /home/username/azure/weblogic-domains/domain1/create-domain-job.yaml
-     /home/username/azure/weblogic-domains/domain1/domain.yaml
-
-   Completed
-   ```
-   {{% /expand %}}
-
-   If your output does not show a successful completion, you must
-   troubleshoot the reason and resolve it before proceeding to the next
-   step.
-
-    {{% notice note %}} This sample creates WebLogic Server pods with reasonable values for memory, CPU, and JVM heap size (as a percentage of memory). These settings were determined by running a skeleton WebLogic domain with minimal or no deployed services and applications on potentially limited or heavily shared container environments. For advice about tuning CPU and memory requests and limits for broader use cases or in a production environment, see the [Pod memory and CPU resources](https://oracle.github.io/weblogic-kubernetes-operator/faq/resource-settings/) FAQ. You can supply different values. Edit `~/azure/weblogic-on-aks/domain1.yaml` and set the desired values for `serverPodMemoryRequest`, `serverPodMemoryLimit`, `serverPodCpuRequest`, `serverPodCpuLimit` and `javaOptions` before running `./create-domain.sh -i ~/azure/weblogic-on-aks/domain1.yaml -o ~/azure -e -v`.
-    {{% /notice%}}
-
-    Here is an excerpt showing reasonable values:
-
-    ```yaml
-    serverPodMemoryRequest: "1.5Gi"
-    serverPodCpuRequest: "250m"
-
-    serverPodMemoryLimit: "1.5Gi"
-    serverPodCpuLimit: "250m"
-
-    javaOptions: -Dweblogic.StdoutDebugEnabled=false -XX:InitialRAMPercentage=25.0 -XX:MaxRAMPercentage=50.0
+- Run command `kubectl get pvc` to get the `pvc name` just created.
```

Comment:
Hi @backwind1233 , it looks like you are creating the PVC manually. In the operator version 4.1, the operator can create the PVC if you provide the name of the `StorageClass` in `domain.spec.configuration.initializeDomainOnPV.persistentVolumeClaim` section. I'm not sure if you are doing this for any specific reason, but just wanted to mention that. You can run the below `kubectl explain domain.spec.configuration.initializeDomainOnPV.persistentVolumeClaim` command to get more details on this schema element or see the documentation [here](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/documentation/domains/Domain.md) .

Reply:
Hi @ankedia 
Thanks for your suggestion, I think we can make use of this new feature.

I gave it a try, and it ran successfully.
I will update the PR soon.

Reply:
> Hi @ankedia Thanks for your suggestion, I think we can make use of this new feature.
> 
> I gave it a try, and it ran successfully. I will update the PR soon.

I have updated the PR.